airbyte-integrations/connectors/source-twilio/source_twilio/streams.py

```diff
@@ -66,13 +76,13 @@ def backoff_time(self, response: requests.Response) -> Optional[float]:
 
         backoff_time = response.headers.get("Retry-After")
         if backoff_time is not None:
-            return float(backoff_time)
-
-    def request_params(
-        self, stream_state: Mapping[str, Any], next_page_token: Mapping[str, Any] = None, **kwargs
-    ) -> MutableMapping[str, Any]:
+        refactor/source-twilio-muttdata
```

Comment:
This is probably a line that got in while fixing conflict, it makes compilation fail.
```suggestion
```